experiments/functional/backup_and_restore/run_litmus_test.yml

```diff
@@ -53,7 +53,7 @@ spec:
             value: "openebs/velero-plugin:ci"
 
           - name: VELERO_VERSION
-            value: "v1.3.0"
+            value: "v1.1.0"
```

Comment:
are we replacing this with gitlab env variable ?

Reply:
Yes